data/xml/N19.xml

```diff
@@ -4599,6 +4609,7 @@
       <url>N19-3002</url>
       <attachment type="presentation">N19-3002.Presentation.pdf</attachment>
       <attachment type="note">N19-3002.Note.pdf</attachment>
+      <video href="https://vimeo.com/347400639" tag="video"/>
     </paper>
     <paper id="3">
```

Comment:
This is a bad match. Maybe the script should print out fuzzy matches so a human can at least skim through them?

Reply:
Looks like a correct match to me.  All matches are either substring or levenshtein matches, so all of them have to be checked.  I could special-case the ones were substring match means perfect match and exclude them from the list the script generates.

Reply:
You’re right. Doing the review in an iPad...